apprentice_teacher.rb

```diff
@@ -1,12 +1,18 @@
-class ApprenticeTeacher
+require_relative "teachers"
+
+class ApprenticeTeacher < Teachers
```

Comment:
An apprentice teacher is a teachers?  Remember that classes should always be named such that the name applies to a single instance of the class.